src/commands/hardis/org/user/freeze.ts

```diff
@@ -93,115 +92,108 @@ export default class OrgUnfreezeUser extends SfdxCommand {
+
+    let apexcode = 'list<userLogin> userLoginList = ['+queryUser+']; \n'+
+    'Set<Id> userIdList = new Set<Id>();\n'+
+    'if(userLoginList != null && userLoginList.size()> 0){\n'+
+        'for(UserLogin userfromList : userLoginList){\n'+
+            'userIdList.add(userfromList.UserId);\n'+
+        '}\n'+
+    '}\n'+
+    'list<User> userList = [SELECT Id,Name,Profile.Name FROM User WHERE Id IN :userIdList];\n'+
+    'system.debug(\'OUTPUTVALUE=\'+JSON.serialize(userList)+\'END_OUTPUTVALUE\'); \n';
+    
     const username = this.org.getUsername(); 
 
-    await fs.readFileSync(path.join(__dirname,'./scripts/apex-freeze.apex'),'utf8');
-    const targetFile = path.join(os.tmpdir(),'apex-freeze.apex');
-    await fs.writeFile(targetFile,query);
-
+    await fs.readFileSync(path.join(__dirname,'./apex-freeze.apex'),'utf8');
+    const targetFile = path.join(__dirname,'apex-freeze.apex');
+    await fs.writeFile(targetFile,apexcode);
+  
     const apexScriptCommand = `sfdx force:apex:execute -f "${targetFile}" --targetusername ${username}`;
-    await execSfdxJson(apexScriptCommand, this, { fail: true, output: true, debug: debugMode });
-    /*const freezeQueryCommand = 'sfdx force:data:soql:query ' + ` -q "${query}"` +
-    ` --targetusername ${username}` + ' --usetoolingapi';*/
-    /*const freezeQueryRes = await execSfdxJson(freezeQueryCommand, this, {
-      output: false,
-      debug: debugMode,
-      fail: true
-    });*/
-
-    //const recordsRaw = freezeQueryRes?.result?.records || freezeQueryRes.records || []
+    const freezeQueryRes = await execSfdxJson(apexScriptCommand, this, { fail: true, output: true, debug: debugMode });
+  
+    let logs = freezeQueryRes?.result?.logs || '' ;
+  
+    let userlistraw= JSON.parse(logs.split('OUTPUTVALUE=')[2].split('END_OUTPUTVALUE')[0]);
     // Check empty result
-    if (true/*recordsRaw.length === 0*/) {
-      const outputString = `[sfdx-hardis] No matching user records found with query ${query}`;
+    if (userlistraw.length === 0) {
+      const outputString = `[sfdx-hardis] No matching user records found for all profile  except ${exceptFilter}`;
       uxLog(this,c.yellow(outputString));
       return { deleted: [], outputString };
     }
 
-    // Simplify results format & display them
-    const records = []/*recordsRaw.map((record: any) => {
+    let userlist = userlistraw.map((record: any) => {
       return {
-        Id: record.Id,
-        isFrozen: record.isfrozen
+        Name: record.Name,
+        Profile: record.Profile.Name
       };
-    });*/
+    });
     uxLog(this,
-      `[sfdx-hardis] Found ${c.bold(records.length)} records:\n${c.yellow(columnify(records))}`
+      `[sfdx-hardis] Found ${c.bold(userlist.length)} records:\n${c.yellow(columnify(userlist))}`
     );
 
-    // Perform update
-   // const udapted = [];
-    //const updateErrors = [];
+    userlistraw = [];
     if (
       !prompt ||
       (await this.ux.confirm(
```

Comment:
not this.ux.confirm, but please use prompts so it can integrate with VsCode :)
Example:

```typescript
    const confirmPush = await prompts({
      type: "confirm",
      name: "value",
      initial: true,
      message: c.cyanBright(
        "Do you want sfdx-hardis to save your configuration on server ? (git stage, commit & push)"
      ),
    });

    if (confirmPush.value === true) {
```